modules/shared/src/main/scala/io/constellationnetwork/schema/priceOracle.scala

```diff
@@ -0,0 +1,18 @@
+package io.constellationnetwork.schema
+
+import io.constellationnetwork.currency.dataApplication.DataUpdate
+import io.constellationnetwork.schema.swap.CurrencyId
+import io.constellationnetwork.schema.transaction.TransactionReference
+
+import derevo.cats.{order, show}
+import derevo.circe.magnolia.{decoder, encoder}
+import derevo.derive
+import eu.timepit.refined.types.numeric.NonNegLong
+
+object priceOracle {
+
+  @derive(decoder, encoder, order, show)
+  case class PricingUpdate(currencyId: CurrencyId, fractionalValue: NonNegLong, markOverride: Boolean, lastReference: TransactionReference)
```

Comment:
This update should be in the AMM itself through the Daemon -- we need the other type committed here from ticket, which should represent the GL0 type (which does not have references or overrides, it's purely an oracle price update (the other declared type from doc.)